project/lang/class.default.php

```diff
@@ -8,26 +8,25 @@ class DefaultLang
 
 	private $default_js = array
 	(
-		'confirmDelete' => "Are you sure you want to delete the task?",
```

Comment:
What's the purpose of your modification ?

Reply:
change names of some features and delete some features that we do not need anymore